FLIR/conservator/local_dataset.py

```diff
@@ -915,21 +915,48 @@ def clone(dataset, clone_path=None, verbose=True, max_retries=5, timeout=5):
 
         return LocalDataset(dataset._conservator, clone_path)
 
+    def validate_annotations(self, index_data):
+        valid = True
+        for frame in index_data["frames"]:
+            target_ids = [a["targetId"] for a in frame["annotations"]]
+
+            # check for duplicate target IDs
+            unique_target_ids = set(target_ids)
+            if len(unique_target_ids) < len(target_ids):
```

Comment:
We're actually going to make duplicate target IDs optional, so I don't think that `cvc validate` should check for this; it will be difficult to maintain in the future.  More complicated "business logic" like this ought to reside in Conservator and be conveyed in in a nice way back to the client

Reply:
Changing strategy for the dupes: going to try to make "push" understand Conservator errors better, and give understandable messages for common reasons for rejecting pushes